### ISSUE TITLE
Privates AddOn oder FriendsOfRedaxo ?

### ISSUE THREAD

**Issue by tbaddade** [TobiasKrais/d2u_helper#11]
Irgendetwas stimmt mit den Credits nicht überein.

<img width="582" alt="Bildschirmfoto 2024-05-15 um 11 38 03" src="https://github.com/TobiasKrais/d2u_helper/assets/338267/6b724a28-77c3-4288-bad6-24e30c63a81f">

**Comment by TobiasKrais** [comment]
Ja, das ist ein Fehler, da das Addon kurzzeitig bei FOR war. Hab ich bei mir im Code schon behoben. Kommt mit dem nächsten Release. Danke für die Meldung.

**Comment by tbaddade** [comment]
> Ja, das ist ein Fehler, da das Addon kurzzeitig bei FOR war. 

Soll das Addon denn später zu For übertragen werden? Denn das Gästebuch bei For zum Bsp. ist von diesem Addon abhängig.

**Comment by TobiasKrais** [comment]
Im Slack Channel wurde eindeutig dafür gestimmt, dass es nicht in FOR landet. Voraussetzung für FOR ist ein spezieller Zweck. Dieses Addon hat viele Zwecke. Evenutell kommen einzelne dieser Zwecke als eigenes Addon nach FOR. Das ist aber Zukunftsmusik.
